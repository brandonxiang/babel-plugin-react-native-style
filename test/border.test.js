const pluginTester = require('babel-plugin-tester');
const reactNativeStyle = require('../src/index');

pluginTester({
  plugin: reactNativeStyle,
  pluginName: 'babel-plugin-react-native-style',
  tests: [
    {
      snapshot: true,
      title: 'border test',
      code:
        "const styles = stylesheet.create({ border: 'solid', borderLeft: '1 solid', borderLeftColor: '#fff' })",
    },
  ],
});
