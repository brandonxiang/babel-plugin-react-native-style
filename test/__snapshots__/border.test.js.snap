// Jest Snapshot v1, https://goo.gl/fbAQLP

exports[`babel-plugin-react-native-style border test: border test 1`] = `
"
const styles = stylesheet.create({ border: 'solid', borderLeft: '1 solid', borderLeftColor: '#fff' })

      ↓ ↓ ↓ ↓ ↓ ↓

import { __RPX } from 'babel-plugin-react-native-style/src/runtime/index';
const styles = stylesheet.create({
  borderWidth: 1,
  borderStyle: 'solid',
  borderColor: '#000',
  borderLeftWidth: 1,
  borderLeftStyle: 'solid',
  borderLeftColor: '#000',
  borderLeftColor: '#fff' });
"
`;
